rfc/0011-connection-string.md

```diff
@@ -47,6 +47,9 @@ _couchbase._tcp.* (for non-secured memcached)
 _couchbases._tcp.* (for SSL-secured memcached)
 
 
+### IPv6 Addresses
+IPv6 addresses will be specified following the IETF RFC-2732 document.  The literal address will be enclosed within "[" and "]" characters to disambiguate a IPv6 octal separator and an explicit port.  Note that IPv4 addresses written in IPv6 notation (i.e. ::192.9.5.5, ::ffff:192.0.2.128) should still be resolved as IPv4 addresses within a client.
```

Comment:
Technically speaking RFC-2732 is about URLs and RFC-5952 is about textual IP address representation.  Since connstr doesn't conform to URL in all cases (the comma one), better add RFC-5952 to the list.

Reply:
I think RFC-5952 is a given here, there is no other method for specifying an IPv6 address that any reasonable (or unreasonable really) person might try to use (note that we don't refer to the IPv4 RFC here either :D).
I specified RFC-2732 as it is relatively generic and only specifies how to encode an IPv6 address for use as a hostname, nothing which conflicts between the URL and Connection Spec RFCs.  I additionally included the most relevant portion of RFC-2732 beneath it to reduce clicks-to-understanding.